validation.js

```diff
@@ -0,0 +1,70 @@
+const form = document.getElementById('form');
```

Comment:
- [ ] The selector you used here is wrong There is no `id` on the form element. There is a class called `'form'`. So I would advise you to use `document.querySelector('.form')`.

Reply:
Hi @Gambit142 
Thank you for your observation. I have added an id on the form element
![Desktop screenshot](https://user-images.githubusercontent.com/29442846/181908430-e26e563f-f85f-4deb-87ac-7a30bf583781.png)
.